oem/ibm/libpldmresponder/file_io_type_pcie.cpp

```diff
@@ -956,8 +975,13 @@ void PCIeInfoHandler::parseTopologyData()
                                                   slot_suffix_location_code;
             slot_final_location_code.push_back(slot_full_location_code);
 
+            // Slot location code structure contains multiple slot location code
+            // suffix structures.
+            // Each slot location code suffix structure is as follows
+            // {Slot location code suffix size(uint8_t),
+            //  Slot location code suffix(variable size)}
             // move the pointer to next slot
-            slot_loc_suf_data += 2;
+            suffix_data += 1 + slot_loccode_suffix_size;
```

Comment:
Please remove the magic number 1

Reply:
I have added comment which explains reason behind adding 1.

Reply:
Adding an explanation to magic number is not considered a good coding practise and error prone with any future changes. Better suggestion is to use explanatory constants.

Reply:
We can provide magic numbers in constants but providing the reason about why specific value is chosen will help developer.